grafana-plugin/src/models/user/user.ts

```diff
@@ -30,6 +30,7 @@ export class UserStore {
   notifyByOptions: any = [];
   currentUserPk?: ApiSchemas['User']['pk'];
   usersCurrentlyBeingFetched: { [pk: string]: boolean } = {};
+  personalWebhook: { webhook: string, context: object };
```

Comment:
I really wasn't sure if I should add this here or if it should be in a `useState` in PersonalWebhookInfo.

Reply:
I think you need to assign the default value `... = {}`, otherwise we experienced that MobX doesn't create an observable